spring-data-jpa-crud-book-dtos/src/main/java/com/luv2code/springboot/cruddemo/rest/customExceptions/BookErrorResponse.java

```diff
@@ -0,0 +1,23 @@
+package com.luv2code.springboot.cruddemo.rest.customExceptions;
```

Comment:
You can use one class for all the error responses right!! No need for each entity.